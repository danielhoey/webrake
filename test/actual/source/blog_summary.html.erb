---
title: Blog Summary
---
<% @source_files.each do |post| %>
  <div>
    <a href="/<%= post.path %>"></a>
    <%= post.content %>
  </div>
<% end %>
