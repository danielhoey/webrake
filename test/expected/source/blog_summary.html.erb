../../actual/source/blog_summary.html.erb