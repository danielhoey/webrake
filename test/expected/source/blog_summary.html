---
title: Blog Summary
---
  <div>
    <a href="/blog/post1.html"></a>
    <p>Blog post 1</p>
  </div>
